bve/src/parse/train_dat/sections/motor.rs

```diff
@@ -0,0 +1,15 @@
+use bve_derive::{FromKVPSection, FromKVPValue};
+
+#[derive(Debug, Default, Clone, PartialEq, FromKVPSection)]
+pub struct MotorSection {
+    #[kvp(bare)]
+    sound_data: Vec<MotorSoundType>,
+}
+
+#[derive(Debug, Clone, PartialEq, FromKVPValue)]
+pub struct MotorSoundType {
+    sound_index: i64,
+    /// Really just sound speed
+    pitch: f32,
```

Comment:
If it's a sound speed and not a real pitch, does it have to be named `pitch`?

Reply:
That's what it's named in the docs. I'm trying to keep it fairly consistent with the docs.